bolt/unittests/Core/BinaryContext.cpp

```diff
@@ -104,6 +104,48 @@ TEST_P(BinaryContextTester, FlushPendingRelocCALL26) {
   EXPECT_FALSE(memcmp(Func2Call, &Vect[12], 4)) << "Wrong forward call value\n";
 }
 
+TEST_P(BinaryContextTester, FlushPendingRelocJUMP26) {
+  if (GetParam() != Triple::aarch64)
+    GTEST_SKIP();
+
+  // This test checks that encodeValueAArch64 used by flushPendingRelocations
+  // returns correctly encoded values for R_AARCH64_JUMP26 relocation for both
+  // backward and forward branches.
+  //
+  // The offsets layout is:
+  // 4:  func1
+  // 8:  b func1
+  // 12: b func2
+  // 16: func2
+
+  char Data[20] = {};
```

Comment:
Test is failing, probably due to using stack array here.

Reply:
thx, updated.